Makefile

```diff
@@ -76,4 +76,7 @@ uninstall: clean
 run: goinstall
 	@${TARGET} &
 
+test:
+	@GOPATH=$(GOPATH) GOBIN=$(GOBIN) go test -v ./...
```

Comment:
Do we still need this inline env `GOPATH` definition? Didn't we get rid of this?

Reply:
Nope. It is removed.